projects/snapshot_bot/config.h

```diff
@@ -170,13 +192,13 @@ class Config
         // **YUCK** why does OpenCV (at least my version) not have a cv::read overload for std::string!?
         cv::String outputPath;
         cv::read(node["outputPath"], outputPath, m_OutputPath.str());
-        m_OutputPath = (std::string)outputPath;
-
-        cv::String testingSuffix;
-        cv::read(node["testingSuffix"], testingSuffix, m_TestingSuffix);
-        m_TestingSuffix = (std::string)testingSuffix;
```

Comment:
Sorry, I should have mentioned this in the PR...

The reason for removing it was that I've changed the test data to also be in an image database format (though there may well not be actual images stored in there) and by default ``ImageDatabase``s are initialised with a timestamp so they're unique. So you'd have e.g. a folder ``training`` with your snapshots in and then your test data would be in ``training/testing/20211117_150400_001`` or whatever. (I think it makes sense to have the testing data as a subfolder in with the training data, but we could also change this.)

It would be sensible to at least the option have more explicit and readable names though. Would it be better to keep them unique (e.g. have a ``testingPrefix`` option) or just have the whole name be specified?

Reply:
ahhh I see - realistically, that probably makes not over-writing your old data rather less error-prone :smile: it might be nice to have an optional prefix or suffix as well though